lib/dialects/sqlite3/query/sqlite-querycompiler.js

```diff
@@ -329,6 +329,13 @@ class QueryCompiler_SQLite3 extends QueryCompiler {
   onJsonPathEquals(clause) {
     return this._onJsonPathEquals('json_extract', clause);
   }
+
+  whereILike(statement) {
+    return `${this._columnClause(statement)} ${this._not(
```

Comment:
can you add test for the not branch too?

Reply:
I'm a bit confused, if I understood correctly the `_not` should add a `not` to the sql statement
to use it with an ilike query with sqlite or any other dialect it has to look like
```js
knex('table').whereNot('col', 'ilike', 'test_string').select();
```
Since there's no a `whereNotILike` function, what the test should be?

Reply:
Is it possible to set that not flag currently at all?